Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -547,6 +558,310 @@ namespace AtomSampleViewer
         ImGui::End();
     }
 
+    void ScriptManager::ShowPrecommitWizard()
```

Comment:
This function is massive. Consider breaking each stage into a different function and using switch statements instead of if else.

Reply:
Done!